docs/concepts/duplicate-legacy-metaos-add-ins.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Manage both a unified manifest and an XML manifest version of your Office Add-in
+description: Learn when and how to maintain versions of your add-in for each type of manifest.
+ms.topic: best-practice
+ms.date: 07/13/2023
+ms.localizationpriority: medium
+---
+
+# Manage both a unified manifest and an XML manifest version of your Office Add-in
+
+Microsoft is making a number of improvements to the Microsoft 365 developer platform. These improvements provide more consistency in the development, deployment, installation, and administration of all types of extensions of Microsoft 365, including Office Add-ins.
+
+One important improvement we're working on is the ability to create a single unit of distribution for all your Microsoft 365 extensions by using the same manifest format: the JSON-formatted [unified manifest for Microsoft 365](../develop/unified-manifest-overview.md).
+
+> [!TIP]
+> For information about converting an existing add-in to use the unified manifest, see [Convert an add-in to use the unified manifest for Microsoft 365](../develop/convert-xml-to-json-manifest.md).
+
+After you've created a version of your add-in that uses the unified manifest, you must ensure the following:
```

Comment:
The phrasing here makes it feel like I have a bunch of tasks to do individually, but there's really just one thing (link them).

Did you consider simplifying to something like:

After you've created a version of your add-in that uses the unified manifest, you must link the existing add-in and the app built using the unified manifest to ensure users never see two copies of the add-in UI inside of Outlook by: 

(then jump to line 31)

Don't remove the existing add-in from AppSource or the Microsoft 365 admin center, or earlier versions of Office will no longer be able to use your add-in.